bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemRegistry.java

```diff
@@ -68,16 +68,6 @@
     public Collection<Item> getItems(String pattern);
 
     /**
-     * Checks whether itemName matches the item name conventions.
-     * Item names must only consist out of alpha-numerical characters and
-     * underscores (_).
-     *
-     * @param itemName the item name to validate
-     * @return true, if the name is valid
-     */
-    public boolean isValidItemName(String itemName);
```

Comment:
Is it safe to remove it? Have you checked that no one in openhab2-addons might e.g. use it?